sort/challenges-12.test.js

```diff
@@ -0,0 +1,372 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named sortBackwards that takes in an array of numbers and returns the same array,
+// with the numbers sorted, highest to smallest.
+// ------------------------------------------------------------------------------------------------
+
+const sortBackwards = (nums) => {
+  return nums.sort((a, b) => {
+    if (a > b) {
+      return -1;
+    } else if (a < b) {
+      return 1;
+    } else {
+      return 0;
+    }
+  });
+};
+
```

Comment:
this can be done in one line, similar to how we sorted the blog articles by date `return nums.sort( (a,b) => b - a );`